### DISCUSSION TITLE
about:blank in Network Tab When Displaying Table on Safari Web Browser

### DISCUSSION THREAD

**Discussion by hci-ikahera** [Budibase/budibase#10896] [Help] [answered]
Hello everyone,
Anyone know what is about:blank everytime we display a table component?
Is there any possibility to remove it?

<img width="1679" alt="Screenshot 2023-06-14 at 14 54 25" src="https://github.com/Budibase/budibase/assets/95213000/8642ab9d-d608-4364-8f5e-366115ce4ac7">

Thank you.

Regards,
Ika

**Comment by andz-bb** [comment, marked as answer]
Hi @hci-ikahera 

The table component is using an iframe with the src attribute set to "about:blank". From what I can tell this is standard practice in cases where you don't want to define a specific source for the iframe. This is working in exactly the same way across all browsers but most other devtools don't display anything for these cases while Safari's logs are more verbose. These about:blank entries in the network request logs are not actually network requests so you can be assured there are no security/privacy/performance concerns related to these. 

Unfortunately I don't think we can prevent these from showing in the Safari network tab at this point. If you have any concerns or questions about this please let me know and I'll try my best to explain it further!

**Reply by hci-ikahera** [reply]
Okay @andz-bb. Thanks for your detail explanation.
